### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by CedrikNikita**
Need to be well tested

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,10 +34,10 @@
     "@aeternity/bip39": "^0.1.0",
     "@aeternity/hd-wallet": "^0.2.0",
     "@stamp/it": "^1.1.0",
-    "@zxing/library": "^0.17.1",
+    "@zxing/library": "^0.18.3",
```
Comment (davidyuk) [inline package.json:37]: There were some breaking changes, I don't think that it can be updated easily

Comment (CedrikNikita) [reply package.json:37]: I have checked it, it works on my Android build.

**Comment by Liubov-crypto** [comment]
Web wallet: 
Generate new wallet - pass
Login into the wallet - pass
Change tokens - pass
Change currency - pass
Send Tip - pass
Claim Tip - pass
Top Up - pass
Withdraw (Send) - pass, via Scan QR code - pass
Activity - pass, sort by date - pass 
Tokens: search - pass
Send - pass
Receive - pass
Names: 
Set name pointer - pass
Extend - pass
Set as default name - pass
Register name - got an error but a name was registered
Invite link - pass
Settings: 
Language - pass
Security&Privacy - pass
Networks - pass
Permission Settings - pass

An empty field in Claimed transaction while I have claimed token: 
claimed

FT design was broken: 
design broken - Invite link
design broken FT
broken design

White stripe in wallet: 
white stripe - web wallet
 
Register name - got an error but a name was registered: 
register a name
test name

I left the wallet for a couple of minutes, and when I returned to check the balance, I received this error:
error when leave a wallet 1
error when leave a wallet
error transactions
token balance
token balance 2

**Comment by Liubov-crypto** [comment]
FF/Chrome extension:
Generate new wallet - pass / pass 
Login into wallet - pass / pass 
Change tokens - pass / pass  
Change currency - pass / pass 
Send Tip - pass / pass  
Claim Tip - pass/ pass 
Top Up - pass/ pass,  connection with Jelly Swap - pass /pass
Withdraw (Send) - pass/pass, via Scan QR code - pass/pass 
Activity - pass /pass, sort by date - pass / pass 
Btw - empty field in Claimed transaction although I have claimed tokens 
Tokens: search - pass /pass
Send - pass /pass
Receive - pass /pass
Names: 
Set name pointer - pass /pass
Extend - pass /pass
Set as default name - pass /pass
Register name - pass/pass 
Invite link - pass/pass
Settings: 
Language - pass/pass
Security&Privacy - pass/pass
Networks - pass/pass
Permission Settings - pass/pass
Remove account - pass/ pass

When I tested, there was a middleware error:
error with mdw

FT design is broken: 
design bug

I registered the name, but then the name disappeared from the wallet:
name

File: package.json (hunk 1 above)
Comment (davidyuk) [reply package.json:37]: this library is used in a web version, `cordova-plugin-qrscanner` is used in Cordova

**Review by davidyuk — APPROVED** [review]
The code changes looks good

**Review by mradkov — APPROVED** [review]

**Pull request merged**